about.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/about.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <title>About Page</title>
+</head>
+<body>
+    <header>
+        <div id="blackHeader">
+            <div class="blackIconsContainer">
+                <a href=""><img class="blackHeaderIcon" src="./images/facebookIcon.png" alt=""></a>
+                <a href=""><img class="blackHeaderIcon" src="./images/twitterIcon.png" alt=""></a>
+            </div>
+            <a href=""><div class="blackHeaderOption">English</div></a>
+            <a href=""><div class="blackHeaderOption">My Page</div></a>
+            <a href=""><div class="blackHeaderOption">Logout</div></a>
+        </div>
+        <div id="whiteHeader">
+            <a href="./index.html"><img id="logoHeader" src="./images/logoOrangeMx-removebg-preview.png" alt="logo"></a>
+            <div id="menuOptions">
+                <a class="menuOptionsLinks" href="./about.html"><div>About</div></a>
```

Comment:
- [ ] After a user navigates to the About Page, this nav link text and href value on line 26 should point to the home page i.e index.html. Since the user is already in the about page, it does not make sense to continue linking to the same about page.